### PR TITLE
Declare `bump2version` as unmaintained

### PR DESCRIPTION
**Pull request by kdeldycke**
Closes:
- https://github.com/c4urself/bump2version/issues/268

### CODE REVIEW

**Comment by florisla** [comment]
I am in favor.  I'll give other people some time to chime in and then merge this.

**Comment by ekohl** [comment]
I'll also say that I don't have time to maintain this anymore and agree people need some supported tool. I would also have accepted the proposal to merge bump-my-version into this repo and grant @coordt permission so users don't have to migrate, but I think that's more up to @c4urself since it's in his GH namespace. Other solutions you can think of are doing some shim that wraps bump-my-version and adds it as a dependency, but I think that may just be postponing the problem.

In short: :+1: for this now, even better would be a smooth migration path for users.
